.circleci/config.yml

```diff
@@ -93,24 +97,37 @@ jobs:
           command:  tox
   Python38-Integration-Tests: 
     docker:
-      - image: cimg/python:3.8
+      - image: cimg/python:3.6
```

Comment:
Why are we starting from 3.6 and building 3.8 instead of starting from 3.8?

Reply:
This was something done while testing which is set back